grug/testing/src/suite.rs

```diff
@@ -243,16 +289,6 @@ where
                 panic!("fatal error while finalizing block: {err}");
             });
 
-        // Sanity check: the number of tx results returned by the app should
-        // equal the number of txs.
-        assert_eq!(
-            num_txs,
-            block_outcome.tx_outcomes.len(),
-            "sent {} txs but received {} tx results; something is wrong",
-            num_txs,
-            block_outcome.tx_outcomes.len()
-        );
```

Comment:
this sanity check is no longer correct, as now PP can insert new txs into the list